### PR TITLE
Add detection for Meteor

### PR DESCRIPTION
**Pull request by Ben305**
This PR adds the detection for Meteor (https://www.meteor.com)

Otherwise, numbro doesn't load the languages automatically

### CODE REVIEW

File: numbro.js [hunk 1]
```diff
@@ -1048,7 +1048,7 @@
         return (typeof process !== 'undefined') &&
             (process.browser === undefined) &&
             process.title &&
-            (process.title.indexOf('node') === 0 || process.title === 'grunt' || process.title === 'gulp') &&
+            (process.title.indexOf('node') === 0 || process.title.indexOf('meteor-tool') > 0 || process.title === 'grunt' || process.title === 'gulp') &&
```
Comment (BenjaminVanRyseghem) [inline numbro.js]: `indexOf('meteor-tool') !== -1` or `indexOf('meteor-tool') >= 0`, isn't it?

Comment (BenjaminVanRyseghem) [inline numbro.js]: (since we check that `node` is the first, you can ignore my comment :smile:)

Comment (Ben305) [inline numbro.js]: Meteor installs itself into the users home folder, .e.g `/Users/Ben/.meteor/...`

That's why `> 0` should be no problem here ;-)

Comment (Ben305) [inline numbro.js]: Can we get this PR and #196 into another release? That would help me a lot.

Comment (Ben305) [inline numbro.js]: @BenjaminVanRyseghem Can we get this into the next release, too?

**Comment by BenjaminVanRyseghem** [comment]
in numbro 1.9.3

**Pull request merged**